src/main/resources/explanations/challenge12.adoc

```diff
@@ -0,0 +1,7 @@
+=== Challenge 12: Docker COPY and WORKDIR
+
+Often, wen convenience containers are made, large parts of the local filesystem of the creator are being copied so that they are available to the author of the convenience container.
```

Comment:
```suggestion
Sometimes large parts of the local filesystem are copied over to the container so that they are available in the container for the convenience of the author.
```